### ISSUE TITLE
Debug modules

### ISSUE THREAD

**Issue by rmorain** [rmorain/kirby#27]
These notebooks fail when running `nbdev_test_nbs`

```
Exception: The following notebooks failed:
01_dataset_builder.ipynb
07_property_scorer.ipynb
08_database_proxy.ipynb
09_create_database_tables.ipynb
11_read_wiki_data.ipynb
13_read_wiki_data.ipynb
```

We need to fix these before we can do most of the other issues. Therefore this issue is a priority. It's also important to make sure the `main` branch passes all the tests.

**Comment by kenneth-vargas** [comment]
I will try to run this tomorrow.

**Comment by kenneth-vargas** [comment]
Test 1 and 8 should pass.

**Comment by rmorain** [comment]
Fixed `08_database_proxy` d882153b34110d175e1b3791ed0369cb887e2ef7

**Comment by rmorain** [comment]
Fixed `01_dataset_builder` da4c5cd

**Comment by rmorain** [comment]
Fixed `07_property_scorer` 8bec210b5d7beff81cc38be8e6c69316228eaa98

**Comment by rmorain** [comment]
Fixed `09_create_database_tables` 57c8c6b6ddacacea352940d6a7f8b16b4554c949

**Comment by rmorain** [comment]
Fixed `10_insert_to_database` 318163b920da5dae2d5a14afc8bdda3e2ed9034b

**Comment by rmorain** [comment]
Fixed `13_read_wiki_data` 0273129008177e7be0f7e0058404de2ca0335c3d

**Comment by rmorain** [comment]
All tests are passing.
